docs/en/ufs/Ozone.md

```diff
@@ -84,6 +84,65 @@ $ ./bin/alluxio fs mount \
   /ozone o3fs://<OZONE_BUCKET>.<OZONE_VOLUME>/
 ```
 
+Possible core-site.xml and ozone-site.xml
+- core-site.xml
+
+```xml
+<configuration>
+  <property>
+    <name>fs.o3fs.impl</name>
+    <value>org.apache.hadoop.fs.ozone.BasicOzoneFileSystem</value>
+  </property>
+  <property>
+    <name>fs.AbstractFileSystem.o3fs.impl</name>
+    <value>org.apache.hadoop.fs.ozone.BasicOzFs</value>
+  </property>
+  <property>
+    <name>fs.defaultFS</name>
+    <value>o3fs://mybucket.myvol</value>
+  </property>
+</configuration>
+```
+
+- ozone-site.xml
+
+```xml
+<configuration>
+  <property>
+    <name>ozone.replication</name>
+    <value>1</value>
+  </property>
+  <property>
+    <name>ozone.metadata.dirs</name>
+    <value>metadata.dir</value>
+  </property>
+  <property>
+    <name>ozone.scm.names</name>
+    <value>localhost</value>
+  </property>
+  <property>
+    <name>scm.container.client.max.size</name>
+    <value>256</value>
+  </property>
+  <property>
+    <name>scm.container.client.idle.threshold</name>
+    <value>10s</value>
+  </property>
+  <property>
+    <name>hdds.ratis.raft.client.rpc.request.timeout</name>
+    <value>60s</value>
+  </property>
+  <property>
+    <name>hdds.ratis.raft.client.async.outstanding-requests.max</name>
+    <value>32</value>
+  </property>
+  <property>
+    <name>hdds.ratis.raft.client.rpc.watch.request.timeout</name>
+    <value>180s</value>
+  </property>
```

Comment:
Again, I'd prefer to have only the necessary properties in these examples - Are any of these actually necessary? I don't know ozone that well, but I assume that the threshold, timeouts, and max requests are unnecessary. Based on my limited knowledge of ozone and my intuition, the only one here that looks like it might be required is `ozone.scm.names`. In that case I'd only like to have just that property here in the example.

If there are any others you think are definitely required, we may keep them in too. I would just like these trimmed as much as possible.

Reply:
Sorry for forgot to explain why these configs are necessary. Because there is a bug in Ozone, some configs without specific key would be set to 0, and the 0 cannot work. This bug related the fellowing configs
```xml
<property>
    <name>scm.container.client.max.size</name>
    <value>256</value>
  </property>
  <property>
    <name>scm.container.client.idle.threshold</name>
    <value>10s</value>
  </property>
  <property>
    <name>hdds.ratis.raft.client.rpc.request.timeout</name>
    <value>60s</value>
  </property>
  <property>
    <name>hdds.ratis.raft.client.async.outstanding-requests.max</name>
    <value>32</value>
  </property>
  <property>
    <name>hdds.ratis.raft.client.rpc.watch.request.timeout</name>
    <value>180s</value>
  </property>
```

Reply:
Thank you - that makes sense. We may leave them in then